extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -472,17 +473,17 @@ private function processInTrackBlogPost() {
 	private function processInTrackBlogComment() {
 		// BADGE_BLOGCOMMENT
 		// is defined check if required because blogs are not enabled everywhere
-		if ( defined( 'NS_BLOG_ARTICLE_TALK' ) && $this->mTitle->getNamespace() == NS_BLOG_ARTICLE_TALK ) {
+		if( defined( 'NS_BLOG_ARTICLE_TALK' ) && $this->mTitle->getNamespace() == NS_BLOG_ARTICLE_TALK ) {
 			// handle only article/comment creating ( not editing )
-			if ( $this->mStatus->value['new'] == true ) {
+			if( $this->mStatus->value[ 'new' ] == true ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1066)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)